selfservice/flow/login/hook.go

```diff
@@ -171,6 +171,8 @@ func (e *HookExecutor) PostLoginHook(w http.ResponseWriter, r *http.Request, g n
 			Debug("ExecuteLoginPostHook completed successfully.")
 	}
 
+	trace.SpanFromContext(r.Context()).AddEvent(events.NewLoginSucceeded(r.Context(), s.ID, i.ID, string(a.Type), string(a.RequestedAAL), a.Refresh, a.Active.String()))
```

Comment:
Would be great to have the info, which OIDC provider ID was used, but I guess that's a bit more complicated?

Reply:
yeah, unfortunately...

Reply:
done!